fluent-plugin-color-stripper.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler", "~> 2.3"
-  spec.add_development_dependency "rake", "~> 10"
-  spec.add_development_dependency "colorize", "~> 0.7.5"
+  spec.add_development_dependency "rake", "~> 13"
+  spec.add_development_dependency "colorize", "~> 0.7"
   spec.add_development_dependency "test-unit", "> 3.2"
-  spec.add_runtime_dependency "fluentd", "~> 0.12"
+  spec.add_runtime_dependency "fluentd", "~> 1.15"
```

Comment:
NOTE: It might be better to support fluentd 1.14.0 even though I no longer use that version.
https://rubygems.org/gems/fluentd/versions/1.14.6

Reply:
Verified that it works.